onnxruntime/test/providers/cpu/tensor/pad_test.cc

```diff
@@ -805,5 +805,28 @@ TEST(PadOpTest, BoolType) {
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
 }
 
+TEST(PadOpTest, ConstantPadAxes) {
```

Comment:
Can we please add more tests?

Reply:
The feature added to Pad(18) is with pad axes input. One new test case was added in ONNX for this new feature. Here the similar unit test is added to test the new feature.